modules/tasks/tasks/src/main/java/org/motechproject/tasks/service/impl/ChannelServiceImpl.java

```diff
@@ -177,6 +181,49 @@ protected void doInTransactionWithoutResult(TransactionStatus status) {
         return channels;
     }
 
+    private void registerLoggerChannel() {
```

Comment:
Since this is a static action, why isn't it defined in a json file? On the very least, if it's registered programatically, it should not happen each time "getAllChannels" is invoked.